tests/install_upgrade_operators/strict_reconciliation/test_hco_default_cpu_model.py

```diff
@@ -152,6 +152,7 @@ def test_set_hco_default_cpu_model_with_existing_vm(
     kubevirt_resource,
     default_vmi_cpu_model,
     updated_vmi_cpu_model,
+    schedulable_nodes,
```

Comment:
`hco_with_default_cpu_model_set` is scope function - to save time - the tests which call it should be under the same class